src/main/java/com/starkbank/utils/Rest.java

```diff
@@ -125,6 +125,14 @@ public static InputStream getContent(Resource.ClassData resource, String id, Str
         return gson.fromJson(jsonObject, (Type) subResource.cls);
     }
 
+    public static <T extends SubResource> List<T> getSubResources(Resource.ClassData resource, String id, SubResource.ClassData subResource, User user, Map<String, Object> options) throws Exception {
+        String content = Response.fetch(Api.endpoint(resource, id) + "/" + Api.endpoint(subResource), "GET", null, options, user).content();
+        JsonObject contentJson = new Gson().fromJson(content, JsonObject.class);
+        JsonObject jsonObject = contentJson.get(Api.getLastName(subResource)).getAsJsonObject();
+        Gson gson = GsonEvent.getInstance();
+        return gson.fromJson(jsonObject, (Type) subResource.cls);
```

Comment:
List<Resource> entities = new ArrayList<>();
for (JsonElement resourceElement : jsonArray) {
    JsonObject jsonObject = resourceElement.getAsJsonObject();
    entities.add(GsonEvent.getInstance().fromJson(jsonObject, (Type) subResource.cls));
};
return entities;